sdw_updater/Updater.py

```diff
@@ -361,12 +352,12 @@ def overall_update_status(results):
 
     if updates_failed:
         return UpdateStatus.UPDATES_FAILED
-    elif reboot_required:
+    if reboot_required:
```

Comment:
(this one's always gonna be weird for me because I feel like the `elif` is more readable and more like defensive coding even though it's not technically necessary. but okay, ruff.)

Reply:
Thanks for pointing that out, it explains why this stuck out at me so much.

Sorry to nitpick, but for the sake of consistency, let's add back the rules that are specifically about readability, so that we have the same standards on all projects. 

So that would be, to my eye 

```
  # superflous-else- rules, find they hurt readability
    "RET505", "RET506", "RET507", "RET508",
  # Find contextlib.suppress() is harder to read
    "SIM105",
    # Find ternary statements harder to read
    "SIM108",
    # Using any()/all() can be harder to read
    "SIM110",   
```

Reply:
Can we re-run the automated fixes now that those rules are suppressed? i.e. back out these changes and elsewhere